stack-19.yaml

```diff
@@ -0,0 +1,13 @@
+resolver: lts-19.6
+allow-newer: true
+packages:
+- http2-grpc-types
+- http2-grpc-proto-lens
+- http2-grpc-proto3-wire
+- warp-grpc
+- http2-client-grpc
+- bench-tool
+extra-deps:
+- proto3-wire-1.2.2
+- git: https://github.com/Concordium/http2-client.git
+  commit: d853c330b25fcb4724b2fe46f98c8d18560010a1
```

Comment:
I think this PR is OK, but we need to update this commit hash after the http2-client one is resolved.

Reply:
updated the reference